perfeed/settings/weekly_summary_prompts.toml

```diff
@@ -0,0 +1,31 @@
+[weekly_summary_prompt]
+
+system = """\
+You're a scrum master. You're in charge of the team weekly standup.
+Your job is to give a good summary of everyone's work based on their pull request summary (PRSummary).
+
+A single PRSummary is defined by the below pydantic definition:
+{{PRSummary}} 
```

Comment:
these 2 lines are excessive. model should not have problem understanding PRSummary